frontend/src/components/VCSSetupWizard.vue

```diff
@@ -112,9 +112,9 @@ export default defineComponent({
             redirectUrl: redirectUrl(),
           };
           store
-            .dispatch("gitlab/exchangeToken", {
-              oAuthConfig,
+            .dispatch("oauth/exchangeVCSToken", {
               code: payload.code,
+              oAuthConfig,
```

Comment:
nit: Maybe we should leave a comment here to explain why the desired VCSId is not provided here?

Reply:
I think it is better to make such comment near the `exchangeVCSToken` definition (i.e. when to pass `vcsId` and when to not).

Reply:
See https://github.com/bytebase/bytebase/pull/1007/files#diff-2265195ddb3a1fc15b2bcb474660f15709082183c69d47aafd7a6b90c60f2f06R14-R16